hazelcast-code-generator/src/main/java/com/hazelcast/client/protocol/generator/CodeGenerationUtils.java

```diff
@@ -56,6 +56,30 @@
         put("com.hazelcast.map.impl.SimpleEntryView", "SimpleEntryView");
     }};
 
+    private static final Map<String, String> JAVA_TO_NODE_TYPES = new HashMap<String, String>() {{
+        put(DATA_FULL_NAME, "data");
+        put("java.lang.String", "string");
+        put("java.lang.Integer", "int32");
+        put("boolean", "boolean");
+        put("int", "int32");
+        put("com.hazelcast.nio.Address", "Address");
+        put("java.util.List", "list");
```

Comment:
We use Arrays for lists. I am not sure how to represent an array here. Because we create a new array as `var a: Array<type> = [];`.